build_cell_ontology_dict.R

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env Rscript 
+
+# This script accepts a collection of (condensed) SDRF files as input and returns a 
+# dictionary containing cell labels as keys and ontology terms as values 
+# for all cells in corresponding SDRF files. In case a cell does not have 
+# ontology mapping, label will map to NA value. 
+
+suppressPackageStartupMessages(require(optparse))
+suppressPackageStartupMessages(require(workflowscriptscommon))
```

Comment:
Could we move any non-essential package imports to until after the arguments have been parsed and checked please?